src/phel/test.phel

```diff
@@ -298,19 +302,21 @@
           :when (get meta :test)]
       (get-in php/$GLOBALS ["__phel" munge-ns fn-name]))))
 
-(defn- run-test [ns]
+(defn- run-test [options ns]
   (let [rt (php/:: \Phel\Runtime\RuntimeSingleton (getInstance))
         _ (php/-> rt (loadNs ns))
-        tests (find-test-fns ns)]
-    (dofor [test :in tests]
+        tests (find-test-fns ns)
```

Comment:
I would do the following here:

1. Change `deftest` to `...(defn ,name {:test true :test-name ,name} ...` (I added `:test-name`)
2. Pass the `filter-options` to `find-test-fns` and do the filtering there. You get the test name with `(get meta :test-name)`

The `BOUND` variable is currently used for internals only (It is used to print better exceptions).